src/microbe_stage/editor/CellEditorComponent.tscn

```diff
@@ -1024,6 +1026,10 @@ layout_mode = 2
 [node name="CompoundStorageStatistics" parent="RightPanel/VBoxContainer/Statistics/VBoxContainer/Body/ScrollContainer/MarginContainer/VBoxContainer" instance=ExtResource("28_vnd4w")]
 layout_mode = 2
 
+[node name="ProcessList" parent="RightPanel/VBoxContainer/Statistics/VBoxContainer/Body/ScrollContainer/MarginContainer/VBoxContainer" instance=ExtResource("41_n2bpg")]
```

Comment:
If I'm looking at this correctly this is just a node you put somewhere? The original idea was that the process panel would be a panel like in the game, so you need to wrap this inside a custom popup and add a button for opening / closing it.

Reply:
It is contained within rightmost editor panel. For me it looks okay

Reply:
There's already a ton of stuff there. I think it is starting to be too much (and especially after my compound storage etc. times are already put there). Especially if you have like at least 8 different processes. So I really think it'd be better to at least initially make this a proper "process panel" like the original issue calls for. This can be further discussed in an UX design context (on dev discord maybe).